docs/querying/aggregations.md

```diff
@@ -432,15 +581,28 @@ This makes it possible to compute the results of a filtered and an unfiltered ag
 
 *Note:* If only the filtered results are required, consider putting the filter on the query itself, which will be much faster since it does not require scanning all the data.
```

Comment:
```suggestion
If only the filtered results are required, consider putting the filter on the query itself, which will be much faster since it does not require scanning all the data.
```